sct-commons/src/main/java/org/lfenergy/compas/sct/commons/util/PrivateUtils.java

```diff
@@ -168,6 +168,18 @@ public static TPrivate createPrivate(List<TCompasTopo> compasTopos) {
         return tPrivate;
     }
 
+    /**
+     * Create Private of given type as parameter
+     * @param compasLNodeStatus type of Private to create
+     * @return created Private
+     */
+    public static TPrivate createPrivateCompasLNodeStatus(String compasLNodeStatus) {
+        TPrivate compasLNodeStatusPrivate = new TPrivate();
+        compasLNodeStatusPrivate.setType("COMPAS-LNodeStatus");
+        compasLNodeStatusPrivate.getContent().add(compasLNodeStatus);
+        return compasLNodeStatusPrivate;
+    }
```

Comment:
Can you make a more generic version with `privateType` as parameter, so we can use this method to create other String privates.
```java
/**
 * Create private containing a simple String
 * @param privateType type attribute of private element to create
 * @param text content of private to create
 * @return the complete private
 */
public static TPrivate createStringPrivate(String privateType, String text) {
    TPrivate tPrivate = new TPrivate();
    tPrivate.setType(privateType);
    tPrivate.getContent().add(text);
    return tPrivate;
}
```